pages/games/[gameType].tsx

```diff
@@ -446,7 +450,7 @@ const Game: NextPage<IGamePageProps> = ({ gameType, version }) => {
       console.log('last message proceess one message', lastMessage)
       const { loser } = lastMessage.message as FinishedGameState;
```

Comment:
Seems that the loser is not in use any more

Reply:
"loser" removed